spec/system/casa_org/edit_spec.rb

```diff
@@ -16,6 +16,7 @@
     stub_twillio
     sign_in admin
     visit edit_casa_org_path(organization)
+    Faker::Nation.unique.clear # Clears used values for Faker::Nation
```

Comment:
Why is this needed?

Reply:
As soon as I added the uniqueness validation, this test started to fail. So I had to make sure that the names generated for the five language records are different. `Faker.unique` was throwing an error. I checked the faker documentation and found this [Ensuring unique values](https://randallreilly.atlassian.net/browse/STRG-903?atlOrigin=eyJpIjoiZWQzN2I0ZDA0NWMzNDAzMDk4NDU2Y2MwMmJjYmQ0ZTkiLCJwIjoiaiJ9).